zoom.sh

```diff
@@ -1,22 +1,26 @@
 #!/usr/bin/env bash
 
+CONF_FILE=$HOME/.zoom/config/zoomus.conf
+
 # Check if the current desktop is using Wayland
 if [[ "$XDG_SESSION_TYPE" == "wayland" ]]; then
+    QPA=wayland-egl
```

Comment:
I think only `wayland` is the appropriate one, even if `wayland-egl` works just fine.

Reply:
Thanks! I changed that.